Composer/packages/extensions/extension/src/hooks/useLgApi.ts

```diff
@@ -12,10 +12,9 @@ export const useLgApi = (shellApi: ShellApi) => {
   const deleteLgTemplates = (lgFileId: string, lgTemplates: string[]) => {
     const normalizedLgTemplates = lgTemplates
       .map((x) => {
-        const lgTemplateRef = LgTemplateRef.parse(x);
-        return lgTemplateRef ? lgTemplateRef.name : '';
+        return LgTemplateRef.parse(x)?.name;
       })
-      .filter((x) => !!x);
+      .filter((x) => x != null) as string[];
```

Comment:
This is technically not the same behavior. 

I would suggest something like `filter(Boolean)`